### ISSUE TITLE
How to modify the optimizer to achieve gradient accumulation

### ISSUE THREAD

**Issue by GeorgePearse** [BinItAI/visdet#89]
**Original issue:** https://github.com/open-mmlab/mmdetection/issues/12280
**Created:** 2024-12-26
**Last updated:** 2024-12-26

---

When reproducing the code, the training accuracy is slightly lower than the paper, and the GPU does not support large batch sizes. How to set gradient accumulation.
